packages/web/src/components/leaving-audius-modal/LeavingAudiusModal.tsx

```diff
@@ -0,0 +1,67 @@
+import { useCallback } from 'react'
+
+import { useLeavingAudiusModal } from '@audius/common'
+import {
+  HarmonyButton,
+  HarmonyButtonType,
+  IconExternalLink,
+  IconInfo,
+  Modal,
+  ModalContent,
+  ModalFooter,
+  ModalHeader,
+  ModalTitle
+} from '@audius/stems'
+
+import { HelpCallout } from '../help-callout/HelpCallout'
+
+import styles from './LeavingAudiusModal.module.css'
+
+const messages = {
+  title: 'Are You Sure?',
+  body: 'This link is taking you to the following website',
+  goBack: 'Go Back',
+  visitSite: 'Visit Site'
+}
+
+export const LeavingAudiusModal = () => {
+  const [{ isOpen, link }, , close] = useLeavingAudiusModal()
+  const handleOpen = useCallback(() => {
+    window.open(link, '_blank', 'noreferrer,noopener')
+  }, [link])
+  const handleClose = useCallback(() => {
+    close()
+  }, [close])
+  return (
+    <Modal
+      bodyClassName={styles.modalBody}
+      isOpen={isOpen}
+      onClose={handleClose}
+    >
+      <ModalHeader>
+        <ModalTitle
+          iconClassName={styles.icon}
+          icon={<IconInfo />}
+          title={messages.title}
+        />
+      </ModalHeader>
+      <ModalContent className={styles.content}>
+        <p className={styles.text}>{messages.body}</p>
+        <HelpCallout icon={<IconExternalLink />} content={link} />
+      </ModalContent>
+      <ModalFooter className={styles.footer}>
+        <HarmonyButton
+          className={styles.button}
+          variant={HarmonyButtonType.GHOST}
+          text={messages.goBack}
+          onClick={handleClose}
+        />
+        <HarmonyButton
```

Comment:
It occurs to me that we could probably use a <Link /> within this button that has the external link properties set on it so that we don't have to use `window.open`. I remember there are cases where window.open isn't allowed...

Reply:
ooh do tell more, I definitely want to avoid the window.open if possible!

Reply:
Just thinking that since `<Link />` supports external link behavior, and we need to open the modal anyway, could embed a `<Link />` to the desired URL within the button, and then just have the button close the modal onClick.
So essentially, just presenting the user with the link again and asking them to click it a second time :-p

Reply:
I like it, but ideally we don't have a link with a button in it or vice versa, but a button _as_ a link